backend/src/main/java/com/woowacourse/momo/storage/service/StorageService.java

```diff
@@ -0,0 +1,57 @@
+package com.woowacourse.momo.storage.service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+public class StorageService {
+
+    private static final String PATH_PREFIX = "./image-save/";
+
+    public String save(MultipartFile file) {
+        File temporary = new File(PATH_PREFIX + file.getOriginalFilename());
+        File directory = new File(PATH_PREFIX);
+
+        fileInit(temporary, directory);
+
+        try (OutputStream outputStream = new FileOutputStream(temporary)) {
+            outputStream.write(file.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        return temporary.getName();
+    }
+
+    private void fileInit(File temporary, File directory) {
+        try {
+            if (!directory.exists()) {
+                directory.mkdirs();
+            }
+            temporary.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
```suggestion
            throw new MomoException(e);
```
`MomoException` 이 아닌 `RuntimeException` 을 던진 이유가 있을까요?

Reply:
당장은 IOException에 대한 처리를 어떻게 해주어야 할지 명확하지 않아 처리하지 않았었는데.. 일단 MomoException으로 던지도록 조치 해 두었습니다~